code/__DEFINES/mobs.dm

```diff
@@ -122,6 +122,13 @@
 #define ENVIRONMENT_SMASH_WALLS 2   //walls
 #define ENVIRONMENT_SMASH_RWALLS 4  //rwalls
 
+
+#define NO_SLIP_WHEN_WALKING	(1<<0)
+#define SLIDE					(1<<1)
+#define GALOSHES_DONT_HELP		(1<<2)
+#define SLIDE_ICE				(1<<3)
+#define SLIP_WHEN_CRAWLING		(1<<4) //clown planet ruin
```

Comment:
:( we can't crawl

Reply:
Soon, I have a PR up for it assuming it goes through then that define will be relevant